lib/src/util/util.dart

```diff
@@ -649,3 +649,33 @@ Future<Permissions> computePermissions(
 
   return computeOverwrites(await computeBasePermissions());
 }
+
+/// Represents a command argument.
+class CommandArgument {
```

Comment:
Couldn't we expose the existing ParameterData type?

Reply:
This is what I thought at first glance, but realized there are some additional fields (like `autocompleteOverride`), which is not necessarily what users want

Reply:
I think it's fine to include that, it's not as if it can do any harm to have that exposed.